ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_submission_service.proto

```diff
@@ -33,6 +33,12 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 service CommandSubmissionService {
 
   // Submit a single composite command.
+  // Errors:
+  // - ``UNAUTHENTICATED``: if the request does not include a valid access token
```

Comment:
Add:
- `RESOURCE_EXHAUSTED`: if the participant or the ledger are overloaded. Clients should back off exponentially and retry.